dlme_airflow/utils/dataframe.py

```diff
@@ -1,27 +1,29 @@
 import os
 import pandas as pd
+import boto3
+from io import StringIO
 
 
 # TODO: If not files are found / dir is empty / etc, this raising an error.
 #       We should handle this error more cleanly.
-def dataframe_from_file(driver: str, data_file_path: str) -> pd.DataFrame:
-    """Returns existing DLME metadata as a Pandas dataframe based on the
-    type of driver.
+def dataframe_from_file(collection) -> pd.DataFrame:
+    """Returns existing DLME metadata as a Pandas dataframe from S3
 
-    @param -- driver The registered DataSource name
-    @param -- existing_dir
+    @param -- data_path
     """
-    for driver_type in ["csv", "json"]:
-        if driver.endswith(driver_type):
-            read_func = getattr(pd, f"read_{driver_type}")
-            return read_func(data_file_path)
+    data_path = collection.data_path()
+    s3 = boto3.client("s3")
+    bucket = os.getenv("S3_BUCKET")
+    key = f"{data_path}/data.csv"
+    obj = s3.get_object(Bucket=bucket, Key=key)
```

Comment:
Since `dataframe_from_file()` works exclusively with objects in s3 buckets should it be renamed to something like `dataframe_from_s3`?